busted/modules/cli.lua

```diff
@@ -129,6 +129,7 @@ return function(options)
   cli:add_option('--lang=LANG', 'language for error messages', 'en', processOption)
   cli:add_option('--loaders=NAME', 'test file loaders', defaultLoaders, processLoaders)
   cli:add_option('--helper=PATH', 'A helper script that is run before tests', nil, processOption)
+  cli:add_option('--lua=LUA', 'The full path to the lua interpreter busted should run under', nil, processOption)
```

Comment:
Technically, it doesn't have to be the full path, right?  It should still work if Lua was in the $PATH (i.e. `lua51`, `lua52`, `lua53`)